src/components/right-sidebar.tsx

```diff
@@ -26,90 +28,112 @@ export default function RightSidebar({ show }: { show?: boolean }) {
+                    src={placeholder}
+                    alt="Placeholder user"
+                    className="aspect-video"
+                  />
+                  <div className="absolute bottom-0 left-0 right-0 top-0 h-full w-full overflow-hidden bg-accent bg-fixed opacity-0 transition duration-300 ease-in-out hover:opacity-60"></div>
+                </div>
+              </Link>
+              <CardHeader>
+                <CardTitle>Getting started</CardTitle>
+                <CardDescription>
+                  Get started with your research.
+                </CardDescription>
+              </CardHeader>
+            </Card>
+            <Card>
+              <Link href="#" className="cursor-default">
+                <div className="relative max-w-xs overflow-hidden bg-cover bg-no-repeat">
+                  <Image
+                    src={placeholder}
+                    alt="Placeholder user"
+                    className="aspect-video"
+                  />
+                  <div className="absolute bottom-0 left-0 right-0 top-0 h-full w-full overflow-hidden bg-accent bg-fixed opacity-0 transition duration-300 ease-in-out hover:opacity-60"></div>
+                </div>
+              </Link>
+              <CardHeader>
+                <CardTitle>Documentation</CardTitle>
+                <CardDescription>
+                  Learn more about the platform.
+                </CardDescription>
+              </CardHeader>
+            </Card>
+            <Card>
+              <Link href="#" className="cursor-default">
+                <div className="relative max-w-xs overflow-hidden bg-cover bg-no-repeat">
+                  <Image
+                    src={placeholder}
+                    alt="Placeholder user"
+                    className="aspect-video"
+                  />
+                  <div className="absolute bottom-0 left-0 right-0 top-0 h-full w-full overflow-hidden bg-accent bg-fixed opacity-0 transition duration-300 ease-in-out hover:opacity-60"></div>
+                </div>
+              </Link>
+              <CardHeader>
+                <CardTitle>Footprint</CardTitle>
+                <CardDescription>
+                  Research Environmental Impact.
+                </CardDescription>
+              </CardHeader>
+            </Card>
+            <Card>
+              <Link href="#" className="cursor-default">
+                <div className="relative max-w-xs overflow-hidden bg-cover bg-no-repeat">
+                  <Image
+                    src={placeholder}
+                    alt="Placeholder user"
+                    className="aspect-video"
+                  />
+                  <div className="absolute bottom-0 left-0 right-0 top-0 h-full w-full overflow-hidden bg-accent bg-fixed opacity-0 transition duration-300 ease-in-out hover:opacity-60"></div>
+                </div>
+              </Link>
+              <CardHeader>
+                <CardTitle>Survey</CardTitle>
```

Comment:
1. Rename it to "Happy or Not", https://www.happy-or-not.com/en/solution/smiley-terminal/
2. They sue us!
3. ...
4. profit?